src/ERG_Mode.cpp

```diff
@@ -283,7 +297,7 @@ void PowerTable::toLog() {
   SS2K_LOG(POWERTABLE_LOG_TAG, "|%s", oString.c_str());
   oString = "|";
   for (int i = 0; i < POWERTABLE_SIZE; i++) {
-    n = sprintf(buffer, "%4d|", this->powerEntry[i].targetPosition);
+    n = sprintf(buffer, "%8d|", this->powerEntry[i].targetPosition);
     oString += buffer;
   }
   SS2K_LOG(POWERTABLE_LOG_TAG, "|%s", oString.c_str());
```

Comment:
You'll want to change the sprintf to %8d on each of those three lines, or I was thinking your could have "%8d" in a string that's dynamically created (to fit the size of targetPosition) at the top of the logging function.